clang/lib/CIR/Dialect/Transforms/FlattenCFG.cpp

```diff
@@ -137,11 +137,9 @@ class CIRScopeOpFlattening : public mlir::OpRewritePattern<cir::ScopeOp> {
     auto *currentBlock = rewriter.getInsertionBlock();
     auto *remainingOpsBlock =
         rewriter.splitBlock(currentBlock, rewriter.getInsertionPoint());
-    mlir::Block *continueBlock;
-    if (scopeOp.getNumResults() == 0)
-      continueBlock = remainingOpsBlock;
-    else
-      llvm_unreachable("NYI");
+    mlir::Block *continueBlock = remainingOpsBlock;
```

Comment:
We can just get rid of the `remainingOpsBlock` variable and define `continueBlock` directly, right?

Reply:
that's right! should be refactored to just use continueBlock